doc/releases/changelog-dev.md

```diff
@@ -8,6 +8,18 @@
   state vector and a number of shots.
   [(#3720)](https://github.com/PennyLaneAI/pennylane/pull/3720)
 
+* Added the needed functions and classes to simulate an ensemble of Rydberg atoms:
+  * A new `RydbergHamiltonian` class is added, which contains the Hamiltonian of an ensemble of
+    Rydberg atoms.
+  * A new `rydberg_interaction` function is added, which returns a `RydbergHamiltonian` containing
+    the Hamiltonian of the interaction of all the Rydberg atoms.
+  * A new `rydberg_drive` function is added, which returns a `RydbergHamiltonian` containing
+    the Hamiltonian of the interaction between a driving laser field and a group of atoms.
+  [(#3749)](https://github.com/PennyLaneAI/pennylane/pull/3749)
+
+* Added `Operation.__truediv__` dunder method to be able to divide operators.
+  [(#3749)](https://github.com/PennyLaneAI/pennylane/pull/3749)
```

Comment:
What is the motivation for this part?

Reply:
To support `qml.PauliX(0) / 2` instead of `1 / 2 * qml.PauliX(0)`